django_celery_beat/migrations/0005_add_solarschedule_events_choices_squashed_0009_merge_20181012_1416.py

```diff
@@ -1,95 +0,0 @@
-# Generated by Django 2.1.2 on 2018-10-12 14:18
```

Comment:
So you deleted the squashed migration here, but that shipped in 1.4.0.  I'm surprised that it didn't complain about it being in the migrations table later on upgrade, but I guess if it works it works /shrug